changelog/issue-5611-1.md

```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: minor
+reference: issue 5611
+---
+The worker type selection at the create taks page now displays the chosen type with appropriate padding from the left.
```

Comment:
@ahmedDevs Thanks for adding those. Is this one a duplicate? 
Looks like we only need one changelog, as this PR is adressing single issue.

Thanks!

Reply:
Yes, this is a duplicate. The desired one is:
changelog/issue-5611.md

Reply:
great, can you please remove it? One is enough :)

Reply:
Done! Thanks for letting me know:)